### ISSUE TITLE
OSCAL Website: Consolidate About and Learn tabs

### ISSUE THREAD

**Issue by Arminta-Jenkins-NIST** [usnistgov/OSCAL#1769]

**Comment by iMichaela** [comment]
@Arminta-Jenkins-NIST - Is the intend to have the Dependency as a separate issue?

**Comment by Arminta-Jenkins-NIST** [comment]
@iMichaela - good question. Do you think there's enough work to merit another issue, or can we do all the work on this ticket?

**Comment by iMichaela** [comment]
> @iMichaela - good question. Do you think there's enough work to merit another issue, or can we do all the work on this ticket?

I believe this #1769 issue, and other issues that resulted from SPIKE 1, need a deeper analysis. The work involves more than a simple merge-info-from-both-tabs in one. I believe we need to look at the content of each page and document the details of the proposed change, identify each page (README.md) that needs to be modified, and how.
Creating Hackmd tracker for how we mant to move the information AND Hackmd pages for the content we propose to make relocate/split/alter so the information is reviewed by the team and maybe other communication experts before we start making any change to the OSCAL website.

**Comment by aj-stein-nist** [comment]
Thanks for the mockup. I looked! For now, I am going to ask we hold off on the consolidation and add it to Sprint 71 only if we make it through other tasks. Thanks.
